index.yaml

```diff
@@ -3835,9 +3879,9 @@ entries:
       catalog.cattle.io/release-name: vsphere-csi
     apiVersion: v1
     appVersion: 2.2.0
-    created: "2021-08-25T15:50:58.902011206-07:00"
+    created: "2021-09-29T14:12:43.383483308-04:00"
     description: vSphere Cloud Storage Interface (CSI)
-    digest: 8db3385ae16017f9f354a4e746ec44519eeadec8a0452b8a4833b224a7aac404
+    digest: a20bc110bca55c0059658daa7e5d5981ce2fcbbdcd2c957f0941170fa43c0afa
```

Comment:
Why'd the current charts digest change if no changes were made ?  This is for vsphere-csi 100.0.0, which is new for 2.6.0.

Reply:
This is a known issue that the digest value changed each time we run the `make charts` even this chart is not touched.